articles/quickstart/webapp/nodejs/_includes/_login.md

```diff
@@ -52,12 +53,24 @@ app.use(passport.session());
 
 ## Trigger Authentication
 
-Auth0's hosted login page can be used to allow users to log in.
+<%= include('../../../_includes/_login_auth0_hosted_login_page') %>
 
-Add a route called `/login` and use the `env` object to set the **Client ID**, **Domain**, and **Callback URL** for your client. This route will instantiate `auth0.WebAuth` and call the `authorize` method to redirect the user to Auth0's hosted login page.
+<%= include('../../../_includes/_login_embed_lock_widget') %>
+
+Add a route called `/login`. Use the `env` object to set the following properties for your client: 
+* Client ID
+* Domain
+* Callback URL
+
+The route creates an instance of the `auth0.WebAuth` object. Then, the route calls the `authorize` method and redirects the user to the Auth0 hosted login page.
+
+You need to make sure you get an OIDC-conformant response. You can achieve it two ways:
```

Comment:
Again, I have no complete image atm, but this feels like it should be in a more prominent place for the quickstarts that use pipeline2